src/PositionManager.sol

```diff
@@ -217,9 +237,35 @@ contract PositionManager is
         if (nativeBalance > 0) recipient.safeTransferETH(nativeBalance);
     }
 
+    /// @param params an encoding of Currency, address
+    function _sweepERC20To(bytes calldata params) internal {
```

Comment:
i think using CurrencyLib this should be easy to make into a `_sweepCurrencyTo` and then you can just do `CurrencyLib.balanceOfSelf` and CurrencyLib.transfer`

Reply:
(and then you can remove that eth sweep from the settle function...)